Modules/@babylonjs/react-native/windows/scripts/PRSetup.ps1

```diff
@@ -0,0 +1,16 @@
+Import-Module $PSScriptRoot\Utils.psm1
+
+# windows build agents don't support the path lengths required for initializing arcore dependencies,
+# so we manually initialize the submodules we need here.

```

Comment:
Ok, I think this answers my earlier question about the submodule update. Instead of having an inclusion list like this, could we have an exclusion list? That might make it easier to maintain. Seems like this is possible? https://stackoverflow.com/a/52185169/7318670

Reply:
updated to match suggestion